pygmt/tests/baseline/test_basemap_polar.png.dvc

```diff
@@ -1,4 +1,4 @@
 outs:
-- md5: 7326eb42c54c6300a634f0723664fb39
-  size: 33205
+- md5: 35573412a627fc004e0107434afec496
+  size: 31804
```

Comment:
Please also remove the `@pytest.mark.xfail` decorator for `test_basemap_polar` in `test_basemap.py` since it's not needed anymore.

Reply:
I'm still getting a failure when I run `test_basemap_polar` after remove the `xfail`. I've confirmed my conda environment has 6.2.0rc2.

Reply:
Hmm, I checkout out this PR's branch and running the tests pass for me (with or without the xfail). What's the error message you're getting?

Reply:
Not sure what I did correct this time, as we have all of the same versions of the dependencies, but I re-generated the polar image and added it to DVC, and everything is passing now :man_shrugging:

Reply:
Fixed in 39061bd